js/src/compatability/compatabilityHelper.js

```diff
@@ -0,0 +1,68 @@
+/* External dependencies */
+import defaults from "lodash/defaults";
+import noop from "lodash/noop";
+
+/* Internal dependencies */
+import DiviHelper from "./diviHelper";
+
+const DEFAULTS = {
+	classicEditorHidden: noop,
+	classicEditorShown: noop,
+};
+
+/**
+ * Adds support (compatability) for Page builders.
+ */
+class CompatabilityHelper {
```

Comment:
I think you have used `compatability` everywhere which seems to be a typo. Can you change it into `compatibility`.